src/robusta/integrations/mail/sender.py

```
@@ -16,42 +19,65 @@
+        
+        # SES configuration
+        self.use_ses = use_ses
+        self.from_email = from_email
+        self.configuration_set = configuration_set
+        self.ses_client = None
+        
+        if use_ses:
+            self._initialize_ses_client(aws_region, aws_access_key_id, aws_secret_access_key)
+    
+    def _initialize_ses_client(self, aws_region: str, aws_access_key_id: Optional[str], 
+                              aws_secret_access_key: Optional[str]):
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)